src/components/FormTodo/FormTodo.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useContext, useState } from 'react';
+import { Todo } from '../../types/types';
+import { TodosContext } from '../../TodosContext';
+
+export const FormTodo: React.FC = () => {
+  const { todos, setTodos } = useContext(TodosContext);
+
+  const [inputValue, setInputValue] = useState('');
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+    if (inputValue.trim() === '') {
```

Comment:
```suggestion
    if (!inputValue.trim()) {
```
Don't compare with an empty string
https://developer.mozilla.org/en-US/docs/Glossary/Falsy